src/Sulu/Bundle/MediaBundle/Media/Storage/ExternalStorageInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\Storage;
+
+interface ExternalStorageInterface extends StorageInterface
+{
+    /**
+     * Returns the public-url for the given file.
+     */
+    public function getPublicUrl(array $storageOption): string;
```

Comment:
Why does this need to be in a separate interface? Wouldn't it be clearer if all `StorageInterface` would have a `getPublicUrl` method? We could probably use the same method to retrieve the URL from the `LocalStorage`. Resp. what are the use cases for this URL afterwards?

Reply:
we could change this to `getType()` (file or redirect) and `getPath()`